tests/enginetests/operator_string_test.go

```diff
@@ -547,3 +547,66 @@ func TestEndsWith(t *testing.T) {
 	}
 }
 
+func TestLastPositionOf(t *testing.T) {
+	tests := []struct {
+		name       string
+		cql        string
+		wantModel  model.IExpression
+		wantResult result.Value
+	}{
+		{
+			name: "LastPositionOfFound",
+			cql: "LastPositionOf('B','ABC')",
+			wantModel: &model.LastPositionOf{
+				BinaryExpression: &model.BinaryExpression{
+					Operands: []model.IExpression{
+						model.NewLiteral("B", types.String),
+						model.NewLiteral("ABC", types.String),
+					},
+					Expression: model.ResultType(types.Integer),
+				},
+			}, 
+			wantResult: newOrFatal(t, 1),
+		},
+		{
+			name: "LastPositionOfNotFound",
+			cql: "LastPositionOf('X', 'ABC')",
+			wantResult: newOrFatal(t, -1),
+		},
+		{
+			name: "LastPositionOfFound2",
+			cql: "LastPositionOf('B', 'ABCDEDCBA')",
+			wantResult: newOrFatal(t, 7),
+		},
+		{
+			name: "LastPositionOfLeftNull",
+			cql: "LastPositionOf(null, 'ABC')",
+			wantResult: newOrFatal(t, nil),
+		},
+		{
+			name: "LastPositionOfRightNull",
+			cql: "LastPositionOf('A', null)",
+			wantResult: newOrFatal(t, nil),
+		},
```

Comment:
Would you mind adding a few cases here?
one where the matcher string is more than one character long
one where the left arg is an empty string
one where the right arg is an empty string